src/lib/ngsi/Metadata.cpp

```diff
@@ -174,3 +174,46 @@ void Metadata::fill(const struct Metadata& md)
   value       = md.value;
   association = md.association;
 }
+
+
+
+/* ****************************************************************************
+*
+* renderV2 - 
+*/
+std::string Metadata::renderV2(ConnectionInfo* ciP, RequestType request, bool comma)
```

Comment:
Actually, ciP is there to make all render() have the same parameters.
This exact function doesn't use it but other renders do.
My idea is to someday have all render methods using the exact same set of params (these three).
Now, if you have an opinion on that idea, then let's talk.